experimental/python/wheel.bzl

```diff
@@ -81,6 +81,37 @@ Sub-packages are automatically included.
     },
 )
 
+def _py_wrapped_input_files_impl(ctx):
+    inputs_to_package  = depset(
+        direct = ctx.files.deps
+    )
+    packageinputfile = ctx.actions.declare_file(ctx.attr.name + '_target_wrapped_inputs.txt')
+    content = ''
+    for input_file in inputs_to_package.to_list():
+        content += _input_file_to_arg(input_file) + '\n'
+    ctx.actions.write(output = packageinputfile, content=content)
+    
+    return [
+        DefaultInfo(
+            files = depset(direct = [packageinputfile])
+        )
+    ]
+
+
+
+py_wrapped_input_files = rule(
```

Comment:
Would it make sense to always create a file with list of files to packaging instead of asking users to make this decision?

Reply:
Yeh I was playing with that before. Let me take another stab at how that feels.